tools/fiptool/tbbr_config.h

```diff
@@ -9,7 +9,7 @@
 
 #include <stdint.h>
 
-#include "uuid.h"
+#include <uuid.h>
```

Comment:
Yes given about comments - assuming we're keeping the groupings then this is fine.